sale_product_set/README.rst

```diff
@@ -11,11 +11,11 @@ list is defined by sales manager.
 This module aims to help salesman to quickly create several sale order lines
 at once in a quotation.
 
-After a *product set* is added to the sale order, each line can be update or
-remove as any other sale order lines.
+After a *product set* is added to the sale order, each line can be updated or
+removed as any other sale order lines.
 
-This is differ to packing products as you don't follow *product set* after it
-was added to the sale order.
+This differs from packing products as you don't follow *product set*
```

Comment:
This sentence makes no sense, perhaps you meant to delete 'as you don't not follow'